packages/plugins/src/foundry/utils/trim.ts

```diff
@@ -0,0 +1,143 @@
+import { BuildInfo } from '@sphinx-labs/core'
+
+export const trimObjectToType = <T>(obj: any, typeTemplate: T): T => {
+  // If the template is either an array then we just return since we don't do any filtering on arrays
+  if (Array.isArray(typeTemplate)) {
+    return obj
+  } else if (typeof typeTemplate !== 'object') {
```

Comment:
if `typeTemplate === null`, this function will error b/c `null` is of `object` type. (It'll error in the for-loop below)